pkg/skaffold/yamlutil/yaml.go

```diff
@@ -0,0 +1,55 @@
+/*
+Copyright 2019 The Skaffold Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package yamlutil
+
+import (
+	"bytes"
+	"io"
+
+	yaml "gopkg.in/yaml.v3"
+)
+
+func UnmarshalStrict(in []byte, out interface{}) error {
+	return unmarshall(in, out, true)
+}
+
+func Unmarshal(in []byte, out interface{}) error {
+	return unmarshall(in, out, false)
```

Comment:
For clarity, could this not just return `yaml.Unmarshal(in, out)` instead of the `unmarshall()` function, or is there other special sauce that I am missing? Perhaps relating to the node function? A comment describing the why of this function name would be helpful.

Reply:
This is exactly same as `yaml3.Unmarshal`. I can add that back.